assets/js/modules/tagmanager/setup.js

```diff
@@ -133,6 +133,18 @@ class TagmanagerSetup extends Component {
 			let errorMsg = '';
 			const responseData = await data.get( TYPE_MODULES, 'tagmanager', 'accounts-containers', queryArgs );
 
+			if ( ! selectedAccount && 0 === responseData.accounts.length ) {
+				data.deleteCache( 'tagmanager', 'list-accounts' );
```

Comment:
This line from the implementation brief is no longer accurate, and likely no longer needed. 
1. `deleteCache` only takes a single argument - the cache key. If needed it should use `invalidateCacheGroup`.  
https://github.com/google/site-kit-wp/blob/4c760c560b89078c712e57e5d037d74b7630806b/assets/js/components/data.js#L356-L363
1. `list-accounts` is not a valid datapoint anymore as of the datapoint refactoring for Tag Manager. I believe the cache to be invalidated here would be ` TYPE_MODULES, 'tagmanager', 'accounts-containers'`
1. I don't think it is necessary to clear this cache since `data.get` does not use cache by default and I don't think this line is currently affecting the behavior.